### PR TITLE
Feature/robot  automation session records cleanup

### PR DESCRIPTION
**Pull request by skristem**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by skristem** [comment]
@davisagli added a sleep for 2 seconds, I couldn't find any other way as it was taking time to update the name of the household.

File: tests/browser/contacts_accounts/add_existingcontact_household.robot [hunk 1]
```diff
@@ -16,5 +16,7 @@ Add Existing Contact to Existing Household
     Populate Lookup Field    Account Name    &{contact1}[LastName] Household
     Click Modal Button    Save
     Click Header Field Link  Account Name
+    ${id} Get Current Record Id
```
Comment (davisagli) [inline tests/browser/contacts_accounts/add_existingcontact_household.robot]: Should be `${id} =`

File: tests/browser/contacts_accounts/add_newcontact_household.robot [hunk 2]
```diff
@@ -9,6 +9,8 @@ Suite Teardown  Delete Records and Close Browser
 Add New Contact to Household With Different LastName 
     &{contact1} =  API Create Contact    Email=skristem@robot.com
     Go To Record Home  &{contact1}[AccountId]
+    ${id} Get Current Record Id
```
Comment (davisagli) [inline tests/browser/contacts_accounts/add_newcontact_household.robot]: Should be `${id} =`

File: tests/browser/contacts_accounts/add_existingcontact_household.robot (hunk 1 above)
Comment (skristem) [reply tests/browser/contacts_accounts/add_existingcontact_household.robot]: this seems to be working, any specific reason why it should be specified that way?

Comment (skristem) [reply tests/browser/contacts_accounts/add_existingcontact_household.robot]: forgot to give extra space and that's why it failed.

Comment (davisagli) [reply tests/browser/contacts_accounts/add_existingcontact_household.robot]: Ohhh that makes sense.

**Pull request merged**